src/test/java/co/rsk/bitcoinj/script/RedeemScriptValidatorTest.java

```diff
@@ -254,6 +254,109 @@ public void hasErpRedeemScriptStructure_whenFlyoverNonStandardErpRedeemScriptRem
         );
     }
 
+    @Test
+    public void hasFlyoverPrefix_whenEmptyFlyoverPrefix_shouldReturnFalse() {
+        Script redeemScript = RedeemScriptUtils.createStandardRedeemScript(defaultRedeemScriptKeys);
+
+        byte[] emptyFlyoverPrefix = {};
+        Script flyoverRedeemScript = RedeemScriptUtils.createFlyoverRedeemScript(
+            emptyFlyoverPrefix,
+            redeemScript
+        );
```

Comment:
if in this situation we want `hasFlyoverPrefix` to return _false_, shouldnt the `createFlyoverRedeemScript` throw an error directly?

Reply:
That method is just a test util. The real builder is in rskj. I'm writing those corner tests for them.